.github/workflows/docker-build.yml

```diff
@@ -194,3 +230,14 @@ jobs:
       chainlink_image_tag: ${{ needs.docker-core-plugins.outputs.docker-manifest-tag }}
       chainlink_version: develop
     secrets: inherit
+
+  call-cre-workflow-load-test:
+    if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+    needs: [ docker-core-plugins-testing ]
+    permissions:
+      contents: read
+    uses: smartcontractkit/chainlink/.github/workflows/cre-workflow-load-tests.yaml@CAPPL-885-workflow-load-test-ci
```

Comment:
Just have to update this ref, then I think it's good from my side.

Reply:
Updated the ref, will update it further after we have observability wired up